lib/between_meals/cmd.rb

```diff
@@ -0,0 +1,38 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+
+# Copyright 2013-present Facebook
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module BetweenMeals
+  class Cmd
+    def initialize(params)
+      @bin = params[:bin] || fail
+      @path = params[:path] || Dir.pwd
```

Comment:
I was starting to object this to because it seems like path here is the path to the binary, but it's not... it's the path to the repo. Just for clarity can we call this repo_path ?